src/rpc/rawtransaction.cpp

```diff
@@ -488,27 +489,44 @@ static UniValue getassetunlockstatuses(const JSONRPCRequest& request)
         throw JSONRPCError(RPC_INTERNAL_ERROR, "No blocks in chain");
     }
 
-    const auto pBlockIndexBestCL = [&]() -> const CBlockIndex* {
-        if (llmq_ctx.clhandler->GetBestChainLock().IsNull()) {
+    std::optional<CCreditPool> poolCL = std::nullopt;
+    std::optional<CCreditPool> poolOnTip = std::nullopt;
+    bool fSpecificCoreHeight = false;
+
+    if (!request.params[1].isNull()) {
+        int nHeight = request.params[1].get_int();
```

Comment:
```suggestion
        const int nHeight{request.params[1].get_int()};
```

Reply:
Actually would be better this to be an optional int, instead of an int + bool flag?